.husky/pre-commit

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npx lint-staged
+# deactivated beause of Git ERROR: .husky/pre-commit: line 5: npx: command not found
+# npx lint-staged
```

Comment:
;(  
What system are you running?
Which node version?

Reply:
Not sure what goes wrong. But it would be great if we can ntroduce such pre-commit kooks this after the miami conference when its a bit more time to finger such things out.

Reply:
All good, understandable

Reply:
is this something to look into again now (after the conference)?